eslint-local-rules.js

```diff
@@ -0,0 +1,10 @@
+const fs = require("fs");
+const path = require("path");
+
+const rulesDir = path.join(__dirname, "scripts", "eslint", "rules");
+const ext = ".js";
+const ruleFiles = fs.readdirSync(rulesDir).filter((p) => p.endsWith(ext));
+
+module.exports = Object.fromEntries(ruleFiles.map((p) => {

```

Comment:
Is this a magic filename for the plugin?

Reply:
`eslint-local-rules.js` is, yeah. Unfortunately, it doesn't allow you to use a dotted name. You can have `eslint-local-rules.js` or `eslint-local-rules/index.js`, and that's it.

I could switch to https://github.com/taskworld/eslint-plugin-local, which has more downloads is literally just `module.exports = require('../../.eslintplugin')`, a better filename.

Reply:
Switched it; it hasn't been updated in 5 years but when it's so simple... :D